setup.py

```diff
@@ -501,7 +502,7 @@ def check_if_rocm_pytorch():
             ],
             extra_compile_args={
                 "cxx": ["-O3"] + version_dependent_macros + generator_flag,
-                "nvcc": append_nvcc_threads(["-O3"] + version_dependent_macros + generator_flag),
+                "nvcc": ["-O3"] + version_dependent_macros + generator_flag,
```

Comment:
@hubertlu-tw Are we not trying to conditionalize the args anymore to maintain the CUDA code? Eg.https://github.com/ROCmSoftwarePlatform/apex/blob/888d164e3dcc88f5b87280033eb32236c5592bb5/setup.py#L132

Reply:
Since there is no additional nvcc args for --transducer extension in upstream Apex, I think we do not need to conditionalize the args for this case.

Reply:
Sorry, in this specific case, I was asking about conditionalizing because I see you removed the `append_nvcc_threads` call wrapping the args. Where is that defined actually and should we not keep it for the CUDA case?